app/controllers/media_controller.rb

```diff
@@ -515,6 +515,17 @@ def fill_reassign_modal
     @no_rights = params[:rights] == 'none'
   end
 
+  def check_annotation_visibility
+    medium = Medium.find_by_id(params[:id])
+    isPermitted = medium.annotations_visible?(current_user)
+    render json: isPermitted
+  end
+  
+  def feedback
```

Comment:
I like `feedback` a little bit more because it's more consistent with the other players (it's not `play_player` or `enrich_player`). Because this is only the name of the method in the media controller (and the feedback button doesn't interfere here), I think there shouldn't be any confusion.